src/lib/config/types/index.ts

```diff
@@ -202,3 +208,18 @@ export const GROUP_RULES_DEFAULT: Configuration['groupRules'] = [
  * - `'_'` is in front of `[a-zA-Z]`.
  */
 export const COMPARE_RULE_DEFAULT: CompareRule = ['_', 'aA'];
+
+export interface OverrideRule {
```

Comment:
Suggest to move it to a separate file, e.g. *override.ts* for future extensions.